cnxeasybake/tests/rulesets/_namespaces.css

```diff
@@ -0,0 +1,41 @@
+@NaMeSpAcE epub 'http://www.idpf.org/2007/ops';
+@namespace xhtml 'http://www.w3.org/1999/xhtml';
+/* The following are unsupported and should give a WARNING */
+@namespace 'http://www.w3.org/1999/xhtml';
+@namespace url('http://www.w3.org/1999/xhtml');
+@namespace foo bar url('http://www.w3.org/1999/xhtml');
+
+p {
+  content: 'Default Works!';
+}
+
+epub|p {
+  content: 'EPUB works';
+}
+
+/* Sort all the list items and move them to #output */
+#input1 > * {
+  move-to: bucket1;
+}
+#output1 {
+  content: pending(bucket1);
+  sort-by: 'xhtml|em';
```

Comment:
Ehwww. So we _are_ going to have to let the namespace ugliness leak out to the recipes? What happens without this? I assume it fails, because no default NS in xpath, right? The alternative would be to parse-as-css and add an explicit default NS (walk the tree, add namespace-selector nodes) serialize, and hand to css->xpath